business_requirement_deliverable_cost/models/business.py

```diff
@@ -122,7 +122,7 @@ class BusinessRequirementDeliverable(models.Model):
     )
 
     @api.multi
-    @api.depends('resource_ids')
+    @api.depends('resource_ids', 'resource_ids.price_total')
```

Comment:
@darshan-serpent This should be part of another PR.

Seems not too big changes and makes sense so at least I would like know if it's documented this Issue or what is about. Could you add some reference?